src/behaviours/cluster/__snapshots__/order-of-sidebar-items.test.tsx.snap

```diff
@@ -359,7 +359,10 @@ exports[`cluster - order of sidebar items when rendered renders 1`] = `
               >
```

Comment:
I think these are left-overs from master. I'll verify and create separate PR for these if so.

Reply:
https://github.com/lensapp/lens/pull/5815